addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -644,8 +669,9 @@ QUnit.test('chat window: close on ESCAPE', async function (assert) {
     assert.verifySteps(['rpc:channel_fold/closed']);
 });
 
-QUnit.test('chat window: state conservation on toggle home menu', async function (assert) {
-    assert.expect(9);
+QUnit.test('[technical] chat window: composer state conservation on toggle home menu', async function (assert) {
+    // technical as show/hide home menu simulation are involved
```

Comment:
Could be a bit more specific, like that home menu implementation details have side-effect on DOM that may make chat window components not work.